lib/ood_core/job/adapters/helper.rb

```diff
@@ -12,6 +12,15 @@ class Helper
         def self.bin_path(cmd, bin_default, bin_overrides)
           bin_overrides.fetch(cmd.to_s) { Pathname.new(bin_default.to_s).join(cmd.to_s).to_s }
         end
+        
+        # Gets a command that submits command on another host via ssh
+        # @param cmd [String] the desired command to execute on another host
+        # @param submit_host [String] where to submit the command
+        # @return [String] command wrapped in ssh if submit_host is present
+        def self.ssh_wrap(cmd, submit_host)
+          return cmd if submit_host.empty?
+          "ssh -t -o BatchMode=yes -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no #{submit_host} \"#{cmd}\""
```

Comment:
See comment below about changing this to an array.

Try using -T instead of -t - you will need to test to confirm.

Also, we do not want the default to be `UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no`. If we want to support this option we need to add a strict_host_checking configuration option to the adapters (default to true) and pass that as an argument to this method. See the LinuxHostAdapter implementation of `ssh_cmd`. We might have a solution https://github.com/OSC/ood-documentation/issues/273 where, when the OnDemand web node is properly configured, strict host checking is always used.

Reply:
-t was used for tty allocation for the interactive apps, but in this case neither -t nor -T are needed